pkgs/stdenv/linux/default.nix

```diff
@@ -375,12 +375,7 @@ in
       targetPlatform = localSystem;
       inherit config;
 
-      preHook = ''
-        # Make "strip" produce deterministic output, by setting
-        # timestamps etc. to a fixed value.
-        commonStripFlags="--enable-deterministic-archives"
```

Comment:
Based on a grep, you can get rid of `commonStripFlags` altogether. (It's defined here and used in pkgs/build-support/setup-hooks/strip.sh`.)

Reply:
Done.